spec/models/info_request_spec.rb

```diff
@@ -1770,16 +1694,18 @@
 
   describe 'when generating a user name slug' do
 
-    before do
```

Comment:
This doesn't seem related to the commit

Reply:
Actually this errors with these changes. I've updated the commit message in e670c9cc7